web/source/osk/visualKeyboard.ts

```diff
@@ -591,7 +592,9 @@ namespace com.keyman.osk {
 
       // Build a layout using the default for the device
       if(typeof layout != 'object' || layout == null) {
-        layout=Layouts.buildDefaultLayout(PVK,kbdBitmask, formFactor);
+        // Using 9.0.0 as a fallback Developer version for keyboards.
+        let kbdDevVersion = utils.Version.parseWithDefault(activeKeyboard['KVER'], "9.0.0");
```

Comment:
Should we define "9.0.0" as a fallback constant in version.ts since you also use it in line 2100?

Reply:
The default version should be moved to `utils.Version` I think, given it is likely to arise again.

Reply:
Along this line - any chance you know offhand when `KVER` was introduced?  I have seen actual `9.0.xxx` values, but nothing earlier.  Not that flawless accuracy on the default actually matters, though.

Reply:
I can't find the specific version it was introduced, sorry. Pre-git days, anyway...